### PR TITLE
Added callback documentation and updated responses to requests.

### PR DESCRIPTION
**Pull request by VizinteanuGeorge**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -182,38 +186,20 @@ print(response.text)
...
-    "nrAudioSeconds": 0,
-    "transcriptionType": "transcription",
-    "filename": "FILE_NAME",
-    "sourceJobId": "sourceJobId" // OPTIONAL, translatedSubtitles only
-  }
-}
-```
-
-> Example B: Response sent when you don't specify a `callbackUrl`:
+> Example A: Response to a POST request:
```
Comment (timobehrens) [inline source/index.html.md]: There is no example B anymore.

File: source/index.html.md [hunk 2]
```diff
@@ -252,12 +240,49 @@ targetLanguage (OPTIONAL) | NONE | `pl `, `en`, `ru`, `fr-ca`, `ca`, `zh`, `ga`,
...
+}
+```
+
 ### Uploading Without `callbackUrl`
-1. When you make a request without the `callbackUrl`, store the value of the `jobId` returned upon a successful call (see `Example B` in right pane).
-2. Use the `jobId` to periodically check the [status](#getting-the-status-of-a-transcription) of the upload request (e.g. every 5 mins).
-   - `status` can either be `OPEN`, `DONE` or `ERROR`.
+
+1. When you make a request without the `callbackUrl`, store the value of the `jobId` returned upon a successful call (
+   see `Example B` in right pane).
```
Comment (timobehrens) [inline source/index.html.md]: There is no example B anymore.

File: source/index.html.md [hunk 3]
```diff
@@ -1002,48 +1048,48 @@ curl --request GET --url 'https://api.amberscript.com/api/jobs?apiKey=YOUR_API_K
...
-        "status": "DONE",
-        "nrAudioSeconds": 191,
-        "transcriptionType": "transcription",
-        "filename": "test.mp4",
-        "jobType": "perfect",
-        "jobOptions": {
-            "transcriptionStyle": "cleanread"
-        },
-        "notes": null
+    "notes": null
```
Comment (timobehrens) [inline source/index.html.md:1069]: Are notes included here?

Comment (VizinteanuGeorge) [reply source/index.html.md:1069]: Yes!

**Pull request merged**